analyzer_test.go

```diff
@@ -94,78 +98,11 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 
 		when("image exists", func() {
 			it.Before(func() {
-				metadata := h.MustReadFile(t, filepath.Join("testdata", "analyzer", "app_metadata.json"))
+				metadata := h.MustReadFile(t, filepath.Join("testdata", "restorer", "app_metadata.json"))
 				h.AssertNil(t, image.SetLabel("io.buildpacks.lifecycle.metadata", string(metadata)))
 				h.AssertNil(t, json.Unmarshal(metadata, &appImageMetadata))
 			})
 
-			it("restores layer metadata", func() {
```

Comment:
Same question as https://github.com/buildpacks/lifecycle/pull/505/files#r582333920

Reply:
> restores app and cache layer metadata
I _think_ that covers the same situation.